### PR TITLE
Update Readme.md

### LINKED ISSUE #90: Finalize Readme.md
To Do: 
-  Add a logo
-  Create a appealing structure for Readme.md
-  Getting Started Guide for developers
-  Add link to video
-  Create User Manual and Sitemap for app and link it to readme.
-  Also update wiki to include important development artifacts

### PR DESCRIPTION
**Pull request by faisal1201**
Final Version of Readme.md for submission unless changes are requested.
Link to demo video needs to be added.

Related Issue: closes #90 #77

### :ballot_box_with_check: Definition of Done checklist
- [x] Meaningful title and valuable description in pull request
- [x] Code change tested manually if not covered by automated tests
- [x] Pull request linked to one or more issues/stories
- [x] Acceptance criteria of the linked stories satisfied
- [x] Code change comprehended by those who approved it
- [x] State of target branch maintained or improved after the code change
- [x] Go to MainActivityTest in the Android App and uncommit @Test <br> Run test manually. Add a picture of the test result in the comments. <br> Comment @Test again to avoid future problems with the CI

### CODE REVIEW

**Comment by Quinstedt** [comment]
The camera is still there as an specification. Why?

**Comment by faisal1201** [comment]
Even though our android app doesn't use the camera stream, by default setup there is a camera in the smart car and code for it in the arduino. But I can remove it since we are not using the camera stream.

**Comment by Quinstedt** [comment]
aha. but then just put an anotation that there is a possibility but we havent used it.

**Review by Dreia — CHANGES_REQUESTED** [review]
I think the Modding instructions should not be there. Rather add the setup instructions page. Because that one has links to all the steps for installing including modding.

File: README.md [hunk 1]
```diff
@@ -1,30 +1,97 @@
...
+* Open Command-line terminal
+* Change Directory to "C:\Program Files\mosquitto"
+* Type "mosquitto -v" to start the broker
+
+### Typical Setup with broker running on background.
+![image](https://user-images.githubusercontent.com/91395562/170592178-e08fefa4-7b16-4a57-b60b-a99bc02a5b40.png)
+
+***
+# Demo Video
+Check out our Demo Video on Youtube "Insert Link" to discover and get a tutorial of what features IslandRush has to offer!
```
Comment (RuthgerD) [inline README.md:85]: I'm guessing the video is not ready yet?

Comment (faisal1201) [reply README.md:85]: You guessed correct

Comment (RuthgerD) [reply README.md:85]: good luck :)

**Comment by faisal1201** [comment]
@Dreia the modding instruction has been removed now

**Review by Quinstedt — APPROVED** [review]
It looks good to me, the changes has been done. The README is updated to the current state of the project

File: README.md [hunk 2]
```diff
@@ -1,30 +1,95 @@
...
-# How does it work?
-The App will be created in Android studios which uses XML for the UI and Kotlin or Java for the code.
-The car-behavior will be writted in C++ using Arduino and the smartCar API.
-We will use the SMCE emulator to visualize the racing track.
-Github will be used for document management and updating the code.
+For more details and clarity,visit our Wiki page on [System Architecture](https://github.com/DIT113-V22/group-08/wiki/System-Architecture).
 
+### SmartCar Specifications
+* ESP32 Board
+* OV767X Camera
```
[the remark this answers is not in the document]
Comment (faisal1201) [reply README.md]: We can remove it.

[the remark this answers is not in the document]
Comment (Quinstedt) [reply README.md]: the optional text is good.

**Review by itsdanesh — DISMISSED** [review]

**Review by Dreia — APPROVED** [review]

**Review by safayoussef — APPROVED** [review]

**Comment by Dreia** [comment]
This issue has been completed.

**Review by Dreia — APPROVED** [review]

**Comment by Quinstedt** [comment]
The video is done, the README is now complete. 
@Dreia @faisal1201 well fought this week !

**Pull request merged**